test/runtests.jl

```diff
@@ -26,6 +26,7 @@ no_ci_dataset_tests = [
     "datasets/vision/cifar100.jl",
     "datasets/vision/emnist.jl",
     "datasets/vision/svhn2.jl",
+    "datasets/manual_datasets.jl"
```

Comment:
let's preserve the file naming convention, this should be `dataset/meshes.jl` or `datasets/meshes/faust.jl`